src/output.rs

```diff
@@ -28,23 +30,27 @@ impl OutputType {
             .or_else(|_| env::var("PAGER"))
             .unwrap_or(String::from("less"));
 
-        let less_path = PathBuf::from(&pager);
+        let pagerflags = shell_words::split(&pager).unwrap_or(vec![pager]);
```

Comment:
`split` will fail if there is a parse error, for example:
```
split("less --option='foo")   # closing quote is missing
```

I don't think that `unwrap_or` would help in such a case?